testing/fuchsia/run_tests.py

```diff
@@ -102,18 +104,31 @@ def bundled_test_runner_of(target_id: str) -> BundledTestRunner:
           tests
       )
   )
+  packages = []
   for test in tests:
-    original_package = test['package']
-    test['package'] = os.path.join(
-        OUT_DIR, test['package'].replace('-0.far', '.far')
-    )
-    try:
-      os.remove(test['package'])
-    except FileNotFoundError:
-      pass
-    os.symlink(original_package, test['package'])
+    if 'package' in test:
+      packages.append(test['package'])
+    else:
+      assert 'packages' in test, \
+             'Expect either one package or a list of packages'
+      packages.extend(test['packages'])
+  resolved_packages = []
+  for package in list(dict.fromkeys(packages)):
+    if package.endswith('-0.far'):
+      # Make a symbolic link to match the name of the package itself without the
```

Comment:
Why is this necessary?

Reply:
In chromium test script, we do not expect the far file name to be renamed to `-0.far` (https://github.com/flutter/engine/blob/a7183373eb5a86ab85852b7a5b077fc73cb4f37f/tools/fuchsia/gen_package.py#L125).
I can definitely add a new parameter in the test script, but it seems not very necessary, using a symbolic link can work it around.